### DISCUSSION TITLE
视频下方仍然有一条黑条，弹幕和视频上方还是缺一点

### DISCUSSION THREAD

**Discussion by jhs114514** [ywmoyue/biliuwp-lite#941] [bug]
### Bug 描述

评论区照片的问题已经解决了，但视频全屏播放的问题还是没解决。为了更直观展现问题，我用手机拍下截图并且标注了地方，希望能有帮助。视频上方缺一点不太容易看出来，把弹幕顶部距离调成0还是看的出来的。可能把视频往下移一点就可以了？我也不太懂。

### 操作步骤

```plain text
打开视频，全屏播放
```

### 期望结果

视频全屏播放能显示全部面积,弹幕完整显示

### 截图

![IMG_20241215_152456_edit_16310695703760](https://github.com/user-attachments/assets/c3460d79-7cea-435b-990d-27450ec2d04e)

### 应用来源

Github/其他Github镜像站

### 安装方式

appx直接安装

### 系统版本

其它 (需注明)

### 其它系统版本

win10 22h2

### 设备类型

笔记本

### 设备架构

x64

### 应用架构

x64

### 代码贡献

我更擅长发现问题

**Comment by ywmoyue** [comment]
@jhs114514 麻烦看下4.7.7.804有没有这个问题

**Comment by jhs114514** [comment]
现在用的是806版本
![屏幕截图 2024-12-16 150202](https://github.com/user-attachments/assets/60efe7df-2d71-4822-b251-522d9325d8c6)

**Comment by ywmoyue** [comment]
@jhs114514 806是基于PR #932 编译的，804是基于dev分支编译的，我需要确认是否该PR引入的问题

**Comment by jhs114514** [comment]
804版本没问题了

**Reply by ywmoyue** [reply]
好的，谢谢
